vega_sim/scenario/registry.py

```diff
@@ -197,4 +198,10 @@
         ext_lock=5 * 60,
     ),
     "parameter_experiment": lambda: ParameterExperiment(),
+    "fuzz_test": lambda: FuzzingScenario(
+        num_steps=2 * 60 * 12,
+        step_length_seconds=30,
```

Comment:
I would probably not bother passing this, it makes the simulation run quite a bit slower and isn't hugely useful if we aren't looking at the results anyway.